ang/civicase/ContactCard.html

```diff
@@ -65,57 +65,57 @@
   </div>
   <!-- End - Additional Contact Dropdown -->
   <script type="text/ng-template" id="contact-popover.html">
-    <div class="civicase__contact-popover">
+    <div class="civicase__contact-popover" ng-init="contact = getContact(contacts[0].contact_id);">
```

Comment:
Lets put it inside controller as this is a part of logic building and not template building and should be moved out.